Dockerfile

```diff
@@ -210,13 +210,36 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y \
   && rustup toolchain install 1.51.0 && rustup default 1.51.0
 USER root
 
+
+### SWIFT
+
+# Install Swift 5.4
+ENV SWIFT_HOME=/opt/swift \
+  PATH="$PATH:${SWIFT_HOME}/bin"
+ARG SWIFT_VERSION=5.4
+RUN apt-get update \
+  && apt-get install -y binutils libc6-dev libedit2 libgcc-9-dev libstdc++-9-dev libz3-dev pkg-config zlib1g-dev
+USER dependabot
+RUN mkdir -p "${SWIFT_HOME}" && chown dependabot:dependabot "${SWIFT_HOME}"
+RUN wget https://swift.org/builds/swift-${SWIFT_VERSION}-release/ubuntu1804/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu18.04.tar.gz \
+  && wget https://swift.org/builds/swift-${SWIFT_VERSION}-release/ubuntu1804/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu18.04.tar.gz.sig \
+  && gpg --verify swift-${SWIFT_VERSION}-RELEASE-ubuntu18.04.tar.gz{.sig,} \
+  && mkdir -p ${SWIFT_HOME}/bin \
+  && tar -xvzf swift-${SWIFT_VERSION}-RELEASE-ubuntu18.04.tar.gz -C ${SWIFT_HOME}/bin \
+  && rm -f swift-${SWIFT_VERSION}-RELEASE-ubuntu18.04.tar.gz \
+  && rm -rf /var/lib/apt/lists/* \
+  && swift --version
```

Comment:
if you can control the `FROM` consider using swift.org published docker images instead of installing the toolchain from tarball

Reply:
otherwise, swift.org will so publish RPMs and Debs which could make this easier